### ISSUE TITLE
Warnings when v1-importing v2-exported topology with distances and NUMA

### ISSUE THREAD

**Issue by rhc54** [open-mpi/hwloc#494]
### What version of hwloc are you using?
v1.11.1

### Which operating system and hardware are you running on?
MacOS 12.0.1, though the data was collected on a Linux system (I don't know the version, but probably something like RHEL 7).

### Details of the problem
Using a topology XML file generated by HWLOC v2.2.0 that describes a Power9 system with multiple NVIDIA GPUs, I converted the file to a v1 XML format by simply exporting it with the `HWLOC_TOPOLOGY_EXPORT_XML_FLAG_V1` flag set. However, when I try to import the resulting v1 file into a v1.11.1 instance, I get the following messages:

```
bad nbobjs
bad nbobjs
****************************************************************************
* hwloc 1.11.11 has encountered what looks like an error from user-given distances.
*
* Group0 (cpuset 0x0000ffff,0xffffffff,0xffffffff,0xffffffff,0xffffffff,0xffffffff nodeset 0x00000101) intersects with Group0 (cpuset 0x00ffffff,0xffffffff,0xffffffff nodeset 0xe0000000,,,,,,,0x00000001) without inclusion!
* Error occurred in topology.c line 1088
*
* Please make sure that distances given through the interface or environment
* variables do not contradict any other topology information.
****************************************************************************
```
My code continues to run and produce what looks like correct results - i.e., when I map-by numa domains, I get what I expected. My question, therefore, is what impact the reported errors have on the resulting topology tree? I'm assuming it just means that the offending objects (in this case, the GPU NUMA domains) are being left out - is that correct?

### Additional information
The two topology files are in the attached tarball:
[p9.topos.gz](https://github.com/open-mpi/hwloc/files/7492407/p9.topos.gz)

**Comment by bgoglin** [comment]
Hello Ralph.

In this specific case, there's no impact on the resulting topology. For some reason, the v1 export is wrong, it contains the distance stuff multiple times, and warnings come from duplicates only. Duplicates are ignored on warning/import, but the original distance matrix seems to be imported correctly.

In the general case, the impact could have been that the distance matrix could not be imported because its parent "Group" object were not inserted, but "normal" objects are unaffected in such cases.

The v1 export bug is still present in git, I'll look at it.

**Comment by rhc54** [comment]
Thanks Brice!

**Comment by bgoglin** [comment]
I fixed the export in master/2.6/2.5, closing this issue. Thanks for the report.
